altair_pandas/test_plotting.py

```diff
@@ -227,3 +227,21 @@ def test_dataframe_area(dataframe, stacked, with_plotting_backend):
     for k, v in {"x": "index", "y": "value", "color": "column"}.items():
         assert spec["encoding"][k]["field"] == v
     assert spec["transform"][0]["fold"] == ["x", "y"]
+
+
+@pytest.mark.parametrize("kind", ["line", "bar", "barh", "area"])
+def test_dataframe_subplot_basic(dataframe, kind, with_plotting_backend):
+    chart = dataframe.plot(kind=kind, subplots=True)
+    spec = chart.to_dict()
+    expected = {"x": "index", "y": "value"}
+    if kind == "bar":
+        assert spec["mark"]["orient"] == "vertical"
+    if kind == "barh":
+        assert spec["mark"]["orient"] == "horizontal"
+        expected["x"], expected["y"] = expected["y"], expected["x"]
+    assert spec["mark"]["type"] == _expected_mark(kind)
+    assert spec["encoding"]["facet"]["field"] == "column"
+    assert spec["columns"] == 1
+    assert spec["transform"][0]["fold"] == ["x", "y"]
+    assert spec["encoding"]["x"]["field"] == expected["x"]
+    assert spec["encoding"]["y"]["field"] == expected["y"]
```

Comment:
Rather than repeating this test here, I think it would be sufficient to parametrize the existing ``dataframe_basic_plot`` test with ("subplots" [True, False]). That has the benefit of testing other combinations as well.

Reply:
Absolutely 😅. Resolved.